### ISSUE TITLE
USWDS-Site: Resolve accessibility failures

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds-site#2688]
## Summary

Resolve the accessibility issues found in the recent [cloud pages beta accessibility scans](https://pages.cloud.gov/sites/379/builds/378612/scans). 

> [!Important]
> This might not be the most recent accessibility scan. Before resolving, we should check the [uswds-site build history](https://pages.cloud.gov/sites/379/builds) to see if a newer scan is available.
